lib/InjectionException.php

```diff
@@ -15,6 +15,51 @@ public function __construct(array $inProgressMakes, $message = "", $code = 0, \E
     }
 
     /**
+     * Add a human readable version of the invalid callable to the standard 'invalid invokable' message.
+     * @param array $inProgressMakes
+     * @param $callableOrMethodStr
```

Comment:
@kelunik Do you really think that a docblock thing like that would actually add any value?

Regardless, I've updated the code to be consistent with how the rest of the library documents params.